code/modules/hydroponics/spreading/spreading_growth.dm

```diff
@@ -11,18 +11,19 @@
 /obj/effect/plantsegment/proc/update_neighbors()
 	// Update our list of valid neighboring turfs.
 	neighbors = list()
-	for(var/turf/simulated/floor in get_cardinal_neighbors())
-		if(spread_distance_limit && get_dist(epicenter, floor) > spread_distance_limit)
+	for(var/turf/simulated/T in get_cardinal_neighbors())
+		if(spread_distance_limit && get_dist(epicenter, T) > spread_distance_limit)
 			continue
-		if(locate(/obj/effect/plantsegment) in floor.contents)
+		if(locate(/obj/effect/plantsegment) in T.contents)
 			continue
-		if(floor.density)
+		if(T.density)
 			if(!isnull(seed.chems[PHENOL]))
-				spawn(rand(5,25)) floor.ex_act(3)
+				spawn(rand(5,25))
+					T.ex_act(prob(80) ? 3 : 2)
```

Comment:
stealth nerf?

Reply:
Doesn't leave a girder 20% of the time, so stealth buff, but I guess it's redundant now that they pass through girders anyways